app/characterization_services/default_characterization_service.rb

```diff
@@ -23,8 +23,12 @@ def characterize(save: true)
     ImagemagickCharacterizationService.new(file_set: file_set, persister: persister).characterize
   end
 
+  def geo_resource?
+    parent.respond_to?(:geo_resource?) && parent.geo_resource?
+  end
+
   def valid?
-    !parent.respond_to?(:geo_resource?) || !parent.geo_resource?
+    !geo_resource?
```

Comment:
Since the media info characterization service is registered after this one, I don't think the media info one will ever run.